README.md

```diff
@@ -37,10 +37,12 @@ For more details refer to the [documentation](https://fourierflows.github.io/Fou
 
 ## Installation
 
-To install, simply do
+To install, use Julia's  built-in package manager (accessed by pressing `]` in the Julia REPL command prompt) to add the package and also to instantiate/build all the required dependencies
 
 ```julia
-] add FourierFlows
+julia>]
+(v1.5) pkg> add FourierFlows
+(v1.5) pkg> instantiate
 ```
```

Comment:
Another possibility is to provide copy/pastable code:

```julia
julia> using Pkg

julia> Pkg.add("FourierFlows")

julia> Pkg.instantiate()
```

Reply:
I guess we opted not to do this because we thought it'd be nice for people to learn how to use the package manager. But then the instructions are more confusing and not copy/pastable...

Reply:
I'll do that in the README and leave the `]` in the Docs... Diversify!